A seguinte tarefa foi executada com sucesso:
<img width="596" alt="Exercicio-Exceptions_01" src="https://github.com/AyrtonFrugoni/Exercicio-Excections/assets/111124974/66ea8444-6fb2-40a3-9965-8eded68a81ef">
  <img width="496" alt="Exercicio-Exceptions_02" src="https://github.com/AyrtonFrugoni/Exercicio-Excections/assets/111124974/a7f767e1-0b3e-40b7-b532-5bdc57047888">
